.circleci/config.yml

```diff
@@ -39,6 +39,14 @@ jobs:
           at: "."
 
       - run: npm run test
+      - store_test_results:
+          path: ./reports/junit
+      - store_artifacts:
+          path: ./reports/junit
+      - run:
+          name: Notify Test Results
+          command: node ./scripts/parseJunit.js
+          when: always
```

Comment:
After the tests have run and the xml file has been saved to the Circle CI container, run the parse script to get the test results. The `when: always` block tells Circle to run this whether the tests pass or fail.